cms/templates/widgets/footer.html

```diff
@@ -14,7 +15,7 @@
         <p>&copy; ${datetime.now(pytz.timezone(settings.TIME_ZONE)).year} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
       </div>
 
-      % if is_any_marketing_link_set(['TOS', 'PRIVACY']):
+      % if is_any_marketing_link_set(['TOS', 'PRIVACY']) or waffle().is_enabled(ENABLE_ACCESSIBILITY_POLICY_PAGE):
         <nav class="nav-peripheral">
```

Comment:
Did the marketing bits simply pass by in the nav when the a11y waffle flag was on and no marketing links? (ie no explosion?)

Reply:
Yes, thankfully! You can take a look [here](https://github.com/edx/edx-platform/blob/master/common/djangoapps/edxmako/shortcuts.py#L70), if you're curious. is_any_marketing_link_set just leverages is_marketing_link_set(name), so the two is_marketing_link_set conditions will just return false.